aui.views/src/AUI/Platform/linux/ADBus.h

```diff
@@ -211,7 +235,7 @@ namespace aui::dbus {
 
             for (const auto&[k, v] : t) {
                 DBusMessageIter item;
-                dbus_message_iter_open_container(&sub, DBUS_TYPE_DICT_ENTRY, fmt::format("{}{}", converter<K>::signature, converter<V>::signature).c_str(), &item);
+                dbus_message_iter_open_container(&sub, DBUS_TYPE_DICT_ENTRY, nullptr, &item);
```

Comment:
Why?

Reply:
DBus accept contained signatures only for VARIANT and ARRAY types, using it with DICT_ENTRY trigger this error:

```
dbus[8850]: arguments to dbus_message_iter_open_container() were incorrect, assertion "(type == DBUS_TYPE_STRUCT && contained_signature == NULL) || (type == DBUS_TYPE_DICT_ENTRY && contained_signature == NULL) || (type == DBUS_TYPE_VARIANT && contained_signature != NULL) || (type == DBUS_TYPE_ARRAY && contained_signature != NULL)" failed in file ../../../dbus/dbus-message.c line 2987.
This is normally a bug in some application using the D-Bus library.
```

And sometimes crashes the app.